scripts/ros-generate-recipes.sh

```diff
@@ -165,20 +142,18 @@ $SUPERFLORE_GEN_OE_RECIPES --dry-run --ros-distro $ROS_DISTRO --output-repositor
 after_commit=$(git rev-list -1 HEAD)
 if [ $after_commit != $before_commit ]; then
     # Identify how the files were generated so that they can be reused.
-    sed -i -e "1 s/\$/ $ROS_DISTRO_RELEASE_DATE $rev/" files/$ROS_DISTRO/cache.yaml
-    sed -i -e "1 s/\$/ $ROS_DISTRO_RELEASE_DATE $rev/" files/$ROS_DISTRO/cache.diffme
```

Comment:
Without (updated versions) of these, the 3 fields that ros-generate-cache.sh added are not retained. But the better way to fix would be to prevent superflore from creating it.